# RNA Metabolism
## RNA Structures and Functions
### Structures
With structures comparable to proteins in complexity and uniqueness.
### Functions
- Function as messengers between DNA and polypeptides (**mRNA**).
- Function as adapters to match a specific amino acid with its specific genetic code carried on mRNA (**tRNA**).
- Function as the structural and catalytic components of ribosomes (**rRNA**).
- Store genetic information in RNA viruses.
- Catalyze the processing of primary RNA transcripts (ribozymes).
## Comparing DNA and RNA synthesis
[Transcription by RNA polymerase in E.*coli*](images/rna_synthesis.jpg)

[Catalytic mechanism of RNA synthesis by RNA polymerase](images/mechanism.jpg)

### Similarities
1. Both are guided by a template;
2. Both have the same polarity in strand extension (5'->3');
3. Both use nucleoside 5’-triphosphates (dNTP or NTP).
### Differences
1. RNA synthesis needs no primers;
2. RNA synthesis only involves a short segment of a large DNA molecule;
3. RNA synthesis uses only one DNA strand as the template ([coding and non-coding strand](images/coding_strand.jpg));
4. RNA synthesis has no proofreading.

## Multiple Functions of RNA polymerase in E.*coli*
The [holoenzyme](images/rna_pol.png) consists of five types of subunits ($\alpha_{2}\beta\beta'\omega\sigma$) and is used to synthesize **all** the RNA molecules in E.*coli*.

The multiple functions includes:

1. searches for initiation sites on the DNA molecule and unwinds a short stretch of DNA (*initiation*);
2. selects the correct NTP and catalyzes the formation of phosphodiester bonds (*elongation*);
3. detects termination signals for RNA synthesis (*termination*).

## RNA Synthesis Features (Not That Important)
- Only a short RNA-DNA hybrid (~8 bp in bacteria) is present through the transcription process.
- At each moment, a region of about 17 bp on the E.*coli* DNA is unwound in the transcription bubble.
- The RNA chain is extended at a rate of 50-90 nucleotides/second by the E.*coli* RNA polymerase.
- Unwinding ahead of and rewinding behind of the transcription bubble produces positive and negative supercoils respectively on the DNA (relieved by the action of **topoisomerases**).
[Changes in the supercoiling of DNA brought by transcription.](images/supercoil.jpg)

## The $\sigma$ subunits enable the E.*coli* RNA polymerase to recognize specific promoter sites
- The RNA polymerase without the $\sigma$ subunit is unable to start transcription at a promoter.
- The $\sigma$ subunit decreases the affinity () of RNA polymerase for general (non-promoter) regions of DNA by a factor of $10^{4}$.
- E.*coli* contains multiple $\sigma$ factors for recognizing different promoters. Each type of $\sigma$ factors allows the cell to coordinately express a set of genes.

## RNA Polymerizing
### [Initiation and Elongation](images/rna_initiation.jpg)
- The enzyme slides to a promoter region and forms a more tightly bound "closed complex".
- Then the polymerase-promoter complex has to be converted to an "open complex", in which a 12-15 bp covering the region from the AT-rich –10 site to +3 site is unwound.
- The essential transition from a "closed" to an "open" complex sets the stage for RNA synthesis, after which the core polymerase moves away from the promoter.
### [Termination](images/rna_termination.jpg)
E.*coli* RNA polymerase stops synthesizing RNA at specific terminator DNA sequences.
- Two classes of transcription terminators have been identified in bacteria: one depends on $\rho$-protein, the other is $\rho$-independent.
- At the $\rho$-independent terminator, the transcribed RNA is able to form a stem-loop (palindromic in DNA sequence) structure followed by a stretch of Us (oligoA in DNA).
- The $\rho$-dependent terminator needs the $\rho$-protein, which has an ATP-dependent RNA-DNA helicase activity, for stopping RNA synthesis.
- The $\rho$-dependent terminator DNA exhibits no obvious sequence similarities (probably the RNA polymerase detects noncontiguous structural features?).
- The $\rho$-dependent terminator is more often found in phages (where it was originally discovered), but rarely in E.*coli*.
- In contrast to what was originally expected, the active signals for stopping RNA synthesis in both $\rho$-independent and $\rho$-dependent transcription terminators lie in the newly synthesized RNA rather than in the DNA template.

## Three types of RNA polmerases in eukaryotic cells
The enzymes are often identified by examining their sensitivity towards $\alpha$-amanitin (from a toxic mushroom).

| Type | Localization | Transcript | Effict towards $\alpha$-amanitin |
| :-:  | :-:          | :-:        | :-:                              |
| RNA pol I | Nucleolus | 18S, 5.8S, 28S rRNA| Insecsitive |
| RNA pol II | Nucleoplasm | mRNA, snRNA | Strongly sebsetive |
| RNA pol III | Nucleoplasm | tRNA, 5S rRNA | Inhibited by high concentrations |

### **RNA pol II**
**RNA polymerase II** (Pol II) binds to promoters of thousands of protein-coding genes.